percy/percy.test.js

```diff
@@ -164,20 +164,27 @@ if (OTP_RR_PERCY_MOBILE) {
     // click the little pattern arrow
     await page.click('#open-route-button-Green')
 
-    await page.waitForSelector('#headsign-selector')
-    const secondPatternOption = await page.$x(
-      "//button[contains(., 'Vine City')]"
-    )
-    await secondPatternOption.click()
-
     await page.waitForSelector('#headsign-selector-label')
+
+    await page.waitForTimeout(500)
+
+    await page.click('#headsign-selector-label')
+
+    await page.waitForTimeout(500)
+
+    await page.keyboard.press('Tab')
+    await page.keyboard.press('Tab')
+    await page.keyboard.press('Tab')
+    await page.keyboard.press('Tab')
+    await page.keyboard.press('Enter')
```

Comment:
Oy vey... are we sure we don't have a better solution here? Or can we at least add a comment about what's going on